src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -13,11 +13,15 @@
 	The microflow timer can be used to execute a microflow on a regular, timed basis. 
 
 */
-dojo.provide("MicroflowTimer.widget.MicroflowTimer");
+define([
+	'dojo/_base/declare',
+	'mxui/widget/_WidgetBase',
+	'dojo/_base/lang'
+], function (declare, _WidgetBase, lang) {
+	'use strict';
 
-mendix.widget.declare("MicroflowTimer.widget.MicroflowTimer", {
+	return declare('MicroflowTimer.widget.MicroflowTimer', [ _WidgetBase ], {

```

Comment:
Dont change the quotes... unless you have to

Reply:
Fixed